### PR TITLE
fix for TIKA-1841 contributed by zetisam

### PR DESCRIPTION
**Pull request by zetisam**

### CODE REVIEW

File: tika-parsers/src/main/java/org/apache/tika/parser/microsoft/HSLFExtractor.java [hunk 1]
```diff
@@ -85,113 +84,106 @@ protected void parse(
...
             if (notes != null) {
                 xhtml.startElement("div", "class", "slide-notes");
 
+                
+                
+                // Notes text
                 textRunsToText(xhtml, notes.getTextParagraphs());
+                
+                // Note headers/footers can be extracted, but content will be duplicate
+                // Repeat the notes footer, if set
```
Comment (chrismattmann) [inline tika-parsers/src/main/java/org/apache/tika/parser/microsoft/HSLFExtractor.java]: commented code should be removed.

**Comment by dstevenson** [comment]
Is there a reason this is being held off on? The current output from XSLF extraction is very hard to parse to do anything useful with since it's a flat structure that may or may not contain a notes block.

**Comment by kkrugler** [comment]
Hi @dstevenson - the last activity on this (via [TIKA-1841](https://issues.apache.org/jira/browse/TIKA-1841) was @chrismattmann requesting an update based on his input (this was back in Aug 2016). I think that's why it stalled out.

I'm guessing it wouldn't be very hard to bring the patch into alignment with the current code base, but it's not mergeable currently.

**Comment by dstevenson** [comment]
@zetisam are you able to merge master to resolve conflicts here? Otherwise we can maybe close this and reopen a new one we have access to

**Comment by zetisam** [comment]
I'll see if I can get to it somewhere in the coming days.
